mappings/net/minecraft/entity/ai/pathing/Path.mapping

```diff
@@ -1,25 +1,41 @@
 CLASS net/minecraft/class_11 net/minecraft/entity/ai/pathing/Path
```

Comment:
Many of these are very wrong. Look at my PR which properly maps these.

Reply:
Is this relevant to this PR? I didn't think I'd changed that.